### PR TITLE
Add CODEOWNERS, push_teams, and the Transactions Postman Collection

### PR DESCRIPTION
**Pull request by npivic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* @tink-ab/npivic
+* @tink-ab/adrinavarro
```
Comment (blazejszumtink) [inline .github/CODEOWNERS]: imo better add a team here

Comment (npivic) [reply .github/CODEOWNERS]: @blazejszumtink Good idea, I'll add a team!

**Review by adrinavarro — APPROVED** [review]

**Review by gusapps — APPROVED** [review]
LGTM

File: Transactions.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"variable": [
+		{
+			"key": "access_token",
```
Comment (gdbroman) [inline Transactions.postman_collection.json:155]: These are some big indentations hehe 👀

Comment (npivic) [reply Transactions.postman_collection.json:155]: Oh lord I didn't touch those :D /blaming someone else... 👯

**Pull request merged**